pytest.ini

```diff
@@ -10,7 +10,6 @@ addopts =
     # --testmon
 
     --durations=10
-    -v
```

Comment:
Revert this. We need to see the details when something breaks.

Reply:
You will see detail if it breaks. "-v with -n auto" produce garbage-like on screen, not something to be trusted.

Reply:
It fulfills its purpose of showing the progress well enough.